### DISCUSSION TITLE
googletest url link is wrong

### DISCUSSION THREAD

**Discussion by tonystarkiss** [deepmodeling/abacus-develop#319] [Show and tell]
the link of googletest in abacus-develop/tests/CMakeLists.txt:
https://github.com.cnpmjs.org/google/googletest.git
should be replaced with https://github.com/google/googletest.git

**Comment by caic99** [comment]
Thank you for your suggestion. We have fixed this in PR #328 .
cnpmjs.org hosts a mirror to github.com , providing better connection to users located in China. However, it is not a best practice to do so.
For issues, we suggest you raising it on issues page. You can also create a PR fixing this issue by yourself. Again, thanks for your contribution!

**Comment by caic99** [comment]
One can download all contents from Bohrium Dataset. Please see: https://github.com/deepmodeling/abacus-develop/blob/develop/toolchain/README.md?plain=1#L54-L66
